core/injector/application_injector.hpp

```diff
@@ -127,8 +127,8 @@ namespace kagome::injector {
     };
 
     // peer info getter
-    auto get_peer_info =
-        [](const auto &injector) -> sptr<libp2p::peer::PeerInfo> {
+    auto get_peer_info = [](const auto &injector,
+                            uint16_t p2p_port) -> sptr<libp2p::peer::PeerInfo> {
```

Comment:
consider creating specialized structures like
```struct P2pPort {
uint16_t port;
};```
and
```struct RpcPort {
uint16_t port;
}```
and injecting them instead of adding more parameters to lambda functions

Reply:
why complicate? not needed here IMO

Reply:
As I wrote, it was only a suggestion. For your consideration.